Fw/Types/StringUtils.cpp

```diff
@@ -29,3 +29,27 @@ U32 Fw::StringUtils::string_length(const CHAR* source, U32 max_len) {
     }
     return length;
 }
+
+I32 Fw::StringUtils::substring_find(const CHAR* source_string, U32 source_size, const CHAR* sub_string, U32 sub_size) {
+
+    FW_ASSERT(source_string != nullptr);
+    FW_ASSERT(sub_string != nullptr);
+
+    for (U32 source_index = 0; source_index < source_size; source_index++) {
```

Comment:
Use `FwIndexType` instead of `U32` for loop indices.

Reply:
Fixed